tests/themes-project/themes/blog_theme/theme.ini

```diff
@@ -0,0 +1,16 @@
+name = "Blog theme"
```

Comment:
I imagine once this is used, most of these fields won't be required. Perhaps the only really required one is line 1? Or maybe we don't even strictly need that since all themes are in their own directory.

Reply:
I don't understand very well your question

Are you referring to the tests? These files are required by the tests because I use the `demo-project/contents` to complete the demo `themes-project`

Are you referring to external themes? There isn't any restriction to models that a theme should contain, only the directory structure and the `theme.ini` file should be mandatory.

Although we could encourage themes to support the Lektor example-project, so changing themes will be easy (because theywill  assume the same basic models)

Reply:
I think he refers to `theme.ini` not needing to be mandatory at all. Is there any info in it that is required for the theme to function?

Reply:
Yes, I totally misunderstood th question, I though he was talking about models

>I imagine once this is used, most of these fields won't be required. 

And yes, the `theme.ini ` is not mandatory at  all at the moment, I just added them as an example

Although this metadata will be useful when adding themes to a repo, and automatically generate a website (or adding to the current website), like [hugo does](https://github.com/gohugoio/hugoThemes)

Maybe the only fields used for Lektor will be `name`, `lektor_minimum_required_version`, and `packages`, but this PR isn't checking any of them, I was thinking to leave all the `theme.ini ` for another PR

Reply:
Maybe the next step is create a themes section to the lektor website to document the expected behavior and encourage some practice when creating a theme

Reply:
Right, I meant to discuss the what will be *necessary* about the `theme.ini` down the road. It was just an aside. I don't think we need to make Lektor use this information in this PR.

Reply:
[bike-shedding] Could we consider using `toml` instead of `ini`, I used `ini` because other configurations files use it, but `toml` has several advantages, like support for lists, and many projects are using it instead of `ini` (pip, cargo(rust))